README.md

```diff
@@ -247,6 +247,16 @@ Here is an example command:
 CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress:interactive
 ```
 
+#### Running e2e outside EU
+**This affects developers based out of the EU (but may affect you if you're using a VPN routing through a country in the EU)**
+
+Running Cypress tests outside the EU will not show the EU consent banners, and this may cause some tests to fail. Set `CYPRESS_SKIP_EU=true` to prevent these tests from failing when outside the EU.
```

Comment:
```suggestion
Running Cypress tests outside the EU will not show the EU consent banners on AMP, and this may cause some tests to fail. Set `CYPRESS_SKIP_EU=true` to prevent these tests from failing when outside the EU.
```

Reply:
> Set `CYPRESS_SKIP_EU=true` to prevent these tests from running when outside the EU.

that doesn't sound right 😖